### ISSUE TITLE
Please update to node/20

### ISSUE THREAD

**Issue by joh-klein** [castastrophe/actions-pr-auto-update#30]
Could you please update this action to use node/20? node/16 actions will stop working soon.

**Comment by castastrophe** [comment]
Yes, this is a great point. I'm pushing up that update now. Thanks for calling that out for me. Sorry for the delay!
